src/components/shared/card.tsx

```diff
@@ -63,10 +71,19 @@ const headingWrapperStyles = css`
     min-height: 150px;
 `
 
-const headingStyles = css`
-    ${headline.xxxsmall()};
-    margin: 0;
+const headingStyles = (itemType: RelatedItemType) => {
+    if (itemType === RelatedItemType.ADVERTISEMENT_FEATURE){
```

Comment:
I think you might have committed to this PR by accident, isn't this for labs content cards?